test/e2e/page-objects/pages/hardware-wallet/connect-hardware-wallet-page.ts

```diff
@@ -57,6 +61,23 @@ class ConnectHardwareWalletPage {
     await this.driver.clickElement(this.connectTrezorButton);
     await this.driver.clickElement(this.continueButton);
   }
+
+  async checkFirefoxNotSupportedIsDisplayed(): Promise<void> {
+    console.log(`Check firefox not supported message is displayed`);
```

Comment:
Very nit: But we capitalize "Firefox" in some other message/error messages:

```suggestion
    console.log('Check "Firefox Not Supported" message is displayed');
```